menu_technical_info/__init__.py

```diff
@@ -0,0 +1,4 @@
+# -*- coding: utf-8 -*-
```

Comment:
remove coding utf-8 line, since it's already the standard for Python3 (everywhere)